dev/AppNotifications/NotificationProperties.cpp

```diff
@@ -147,3 +157,15 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationProperties::get_ActivityId(_Out_ GU
     *activityId = GUID_NULL;
     return S_OK;
 }
+
+STDMETHODIMP_(HRESULT __stdcall) NotificationProperties::get_ToastDevicesData(_Out_ ToastABI::IToastDevicesData** devicesData) noexcept
+{
+    *devicesData = nullptr;
+    auto lock{ m_lock.lock_shared() };
+    if (m_toastDevicesData != nullptr)
```

Comment:
Just do the copy-to - it's safe even if the "source" is null:

https://github.com/microsoft/cppwinrt/blob/f9ec1986083a70d2f99d726b06b00a38cb2d1054/strings/base_com_ptr.h#L194-L230

```c++
{
    auto lock{m_lock.lock_shared()};
    m_toastDevicesData.copy_to(devicesData);
    return S_OK;
}
```

Reply:
Will do.